docs/authoring.md

```diff
@@ -25,41 +25,16 @@ The AsyncAPI generator has been built with extensibility in mind. The package us
    
 ## Filters
 
-A filter is a helper function that you can create to perform complex tasks. They are JavaScript files that register one or many [Nunjuck filters](https://mozilla.github.io/nunjucks/api.html#custom-filters). The generator will parse all the files in the `.filters` directory.
+A filter is a helper function that you can create to perform complex tasks. They are JavaScript files that register one or many [Nunjuck filters](https://mozilla.github.io/nunjucks/api.html#custom-filters). The generator parses all the files in the `.filters` directory. Functions exported from these files are registered as filters.
 
-Each file must export a function that will receive the following parameters:
-
-* `Nunjucks`: a reference to the [Nunjucks](https://mozilla.github.io/nunjucks) template engine used internally by the generator.
-* `_`: a convenient [Lodash](https://www.lodash.com) reference.
-* `Markdown`: a reference to the [markdown-it](https://github.com/markdown-it/markdown-it) package. Use it to convert Markdown to HTML.
-* `OpenAPISampler`: a reference to the [openapi-sampler](https://github.com/Redocly/openapi-sampler) package. It generates examples from OpenAPI/AsyncAPI schemas.
-
-The common structure for one of these files is the following:
-
-```js
-module.exports = ({ Nunjucks, _, Markdown, OpenAPISampler }) => {
-  Nunjucks.addFilter('yourFilterName', (yourFilterParams) => {
-    return 'doSomething';
-  });
-};
-```
-
-An example filter called `camelCase` which uses the Lodash function `_.camelCase(String)` to convert a template string as camel case:
-
-```js
-module.exports = ({ Nunjucks, _, Markdown, OpenAPISampler }) => {
-  Nunjucks.addFilter('camelCase', (str) => {
-    return _.camelCase(str);
-  });
-};
-```
-
-And then you can use the filter in your template as follows:
+You can use the filter function in your template as in the following example:
 
 ```js
 const {{ channelName | camelCase }} = '{{ channelName }}';
 ```
 
+In case you have more than one template and want to reuse filters, you can put them in a single library. You can configure such a library in the template configuration under `filter` property. You can also use the official AsyncAPI [filters library](https://github.com/asyncapi/generator-filters). To learn how to add such filters to configuration [read more about the configuration file](#configuration-file).
```

Comment:
```suggestion
In case you have more than one template and want to reuse filters, you can put them in a single library. You can configure such a library in the template configuration under `filters` property. You can also use the official AsyncAPI [filters library](https://github.com/asyncapi/generator-filters). To learn how to add such filters to configuration [read more about the configuration file](#configuration-file).
```